src/applications/merchant-app/MerchantPopulateItemsTab.svelte

```diff
@@ -291,4 +346,9 @@
     }
   }
 
-</style>
\ No newline at end of file
+  .fix-height-checkbox {
+    height: 20px; /* Foundry checkbox are 20px.

```

Comment:
I found that 20px is coincidence, this checkbox has redefined height but overall div height is still 20px and it's in px, not rem, so wont be changed.